adding an about page (#139)
jupyterhub/binder: doc/about.rst

.. _about:

======================
About ``mybinder.org``
======================

> This page explains some of the teams and organizations behind mybinder.org.

Reviewer: I like that this starts with "teams and orgs behind mybinder.org" so I would swap the order of the sections below and put the "tech" one at the end. Right now it is a bit weird to talk about humans and then lead with tech :)

Reviewer 2: Agree with @betatim

What technology runs mybinder.org?
==================================

The technology behind mybinder.org is primarily composed of three open-source projects:

* `JupyterHub <https://z2jh.jupyter.org>`_, which manages cloud infrastructure for user instances
* `repo2docker <repo2docker.readthedocs.io>`_, which builds Docker images from GitHub repositories
* `BinderHub <binderhub.readthedocs.io>`_, which orchestrates the above two projects and
  provides the Binder interface.

Each of these are open and inclusively-governed projects. Currently, these are all officially

> hosted as a part of Project Jupyter, an open project that creates open tools for data science

Reviewer 2: Link to Project Jupyter governance page on jupyter.org

Reviewer 2: I would start with org definition, teams, sponsors (Moore) and end with technical projects. 

Perhaps make  the distinction that all  of the technical components are governed by Project Jupyter under the JupyterHub org. The long term sustainability of mybinder.org is driven by the mybinder.org team who is made up of ...

infrastructure and interactive computation. The mybinder.org team is

> heavily involved in each, though mybinder.org does not need to have a controlling

Reviewer 2: This is a bit confusing. I would strike the sentence.

stake in any of them.

Who runs mybinder.org?
======================

> mybinder.org is a single deployment of the BinderHub technology. It is run as a free, public

Reviewer: ```suggestion
mybinder.org is a deployment of the BinderHub technology. It is run as a free, public
```

Reviewer 2: remove free and just say public

> service to the scientific and educational community. It also serves as a way for the Binder

Reviewer: ```suggestion
service to the scientific and educational community. It is used by the Binder
```

> service to the scientific and educational community. It also serves as a way for the Binder

Reviewer: Maybe not explicitly limit who it is for? For example do we mean to exclude not-for-profits? I don't have a good alternative formulation but maybe something like "... to all those who want to share their work publicly"?

> project to test the "cutting edge" of Binder and demonstrate the technology.

Reviewer: Move this paragraph to L6 as it describes what mybinder.org is, should lead with that (and doesn't really go in the "who runs this" section)

> project to test the "cutting edge" of Binder and demonstrate the technology.

Reviewer: ```suggestion
project to demonstrate the "cutting edge" of its technology.
```

Currently, the mybinder.org team is comprised of the `Binder Project team <https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html#binder-team>`_.
However, as time moves on we expect that these two teams will start to diverge as more
institutions deploy BinderHub for their own purposes, and want to help govern the direction

> of the project.

Reviewer: I'd remove this paragraph from this document and put it in a document on how to join the Binder project. If you deploy your own BinderHub you don't end up involved with mybinder.org. You can however be involved in running mybinder.org without deploying your own BinderHub. If you deploy your own Hub you probably want to become part of the Binder project to help set direction in which the team goes. But maybe you are happy to be another contributor to the software projects (BinderHub and repo2docker) without being involved in the Binder project.

Author: cool, removed it in the latest push. WDYT? I'll hold off on adding a "how you can get involved" page to keep this PR scoped more cleanly

What kind of an organization is the mybinder.org team?
======================================================

> The mybinder.org team is currently housed as a member of Project Jupyter, which is

Reviewer: I think we have a few too many different names for the same team. There is Binder team, Binder project, mybinder.org team :-/

How about using (and then we need to try and get it uniformly applied over this page): mybinder.org is operated by the Binder project (an organisation) which is part of Project Jupyter. The term "Binder team" refers to the humans that make up the Binder project. BinderHub and repo2docker (as an example) are software projects that the Binder team leads and contributes to.

There is no mybinder.org team, Binder project team, BinderHub team or other variations.

Author: ok, latest push tries to clean some of that up, what do folks think?

itself housed under NumFocus, a 501c3 non-profit. 

Who pays for mybinder.org?
==========================

> Currently, the BinderHub deployment at mybinder.org is paid for with a

Reviewer 2: instead of paid perhaps funded.
